guides/introduction/installation.md

```diff
@@ -0,0 +1,150 @@
+
+  ```sh
+  mix archive.install hex phx_new
+  ```
+
+3. Generate a new Phoenix application
+
+  ```sh
+  mix phx.new --install admin
+  ```
+
+4. Add `:beacon_live_admin` dependency to `mix.exs`
+
+  ```elixir
+  {:beacon_live_admin, github: "BeaconCMS/beacon_live_admin"}
+  ```
+  
+5. Add `:beacon_live_admin` into `:import_deps` in file `.formatter.exs`
+
+6. Run `mix setup`
+
+7. Add `beacon_live_admin` to your application `router.ex` file:
+
+  ```elixir
+  use Beacon.LiveAdmin.Router # <- add this line
+  
+  pipeline :browser do
+    # ...
+    # ommited for brevity
+    plug Beacon.LiveAdmin.Plug # <- add this line
+  end
+
+  # add the following scope
+  scope "/admin" do
+    pipe_through :browser
+    beacon_live_admin "/" 
+  end
+  ```
+
+Now you can follow the guide [your first site](https://github.com/BeaconCMS/beacon/blob/main/guides/introduction/your_first_site.md) to setup your first layout, pages, and components.
+
+## Detailed Instructions
+
+### Elixir 1.13 or later
+
+The minimum required version to run Beacon is Elixir v1.13. Make sure you have at least that version installed along with Hex:
+
+1. Check Elixir version:
+
+```sh
+elixir --version
+```
+
+2. Install or update Hex
+
+```sh
+mix local.hex
+```
+
+If that command fails or Elixir version is outdated, please follow [Elixir Install guide](https://elixir-lang.org/install.html) to set up your environment correctly.
+
+### Phoenix 1.7 or later
+
+Beacon also requires at least Phoenix v1.7 to work properly, make sure you have the latest `phx_new` archive - the command to generate new Phoenix applications.
+
+```sh
+mix archive.install hex phx_new
+```
+
+### Generate a new application
+
+We'll be using `phx_new` to generate a new application. You can run `mix help phx.new` to show the full documentation with more options, but let's use the default values for our new site:
+
+```sh
+mix phx.new --install admin
+```
+
+Or if you prefer an Umbrella application, execute:
+
+```sh
+mix phx.new --umbrella --install admin
+```
+
+Beacon supports both.
+
+After it finishes you can open the generated directory: `cd admin`
+
+### Install Beacon LiveAdmin
+
+1. Edit `mix.exs` to add `:beacon_live_admin` as a dependency:
+
+```elixir
+{:beacon_live_admin, github: "BeaconCMS/beacon_live_admin"},
+```
+
+Or add to `admin_web` if running in an Umbrella app.
+
+2. Add `:beacon_live_admin` to `import_deps` in the .formatter.exs file:
+
+```elixir
+[
+ import_deps: [:ecto, :ecto_sql, :phoenix, :beacon_live_admin],
+ # rest of file ommited
+]
+```
+
+3. Run `mix setup`
+
+### Mount admin in the router
+
+Beacon LiveAdmin requires calling a plug in the pipeline and calling the `beacon_live_admin` macro to mount the user interface. It will look like this:
+
+  ```elixir
+  use Beacon.LiveAdmin.Router # <- add this line
+  
+  pipeline :browser do
+    # ...
+    # ommited for brevity
+    plug Beacon.LiveAdmin.Plug # <- add this line
+  end
+
+  # add the following scope
```

Comment:
I think it might be worthwhile to mention that this scope should be above Beacon. Otherwise, it won't be reachable due to the catch-all route set up by Beacon. 

Good work, btw 👍